Cargo.toml

```diff
@@ -57,7 +57,7 @@ proptest = { version = "1.5.0", default-features = false, features = ["std"] }
 rand = "0.8.5"
 reqwest = { version = "0.12.4", default-features = false }
 rmp-serde = "1.3.0"
-ruma = { git = "https://github.com/ruma/ruma", rev = "b266343136e8470a7d040efc207e16af0c20d374", features = [
+ruma = { git = "https://github.com/ruma/ruma", rev = "5c9ef9f425d8f8b330e46d215c683bf799a28d50", features = [
```

Comment:
Maybe a comment should be added to only update Ruma using the [`ruma-0.12`](https://github.com/ruma/ruma/tree/ruma-0.12) branch, to make sure no one tries to update using the `main` branch before we plan to make a breaking release?

Reply:
That's a good idea.